R/generic.R

```diff
@@ -7,7 +7,7 @@
 #'
 #' @param name The name of the generic. This should be the same as the object
 #'   that you assign it to.
-#' @param signature A character vector providing the names of arguments to
+#' @param dispatch_args A character vector providing the names of arguments to
 #'   dispatch on. If omitted, defaults to the required arguments of `fun`.
```

Comment:
Does everyone know that "the required arguments of `fun`" `==` "the arguments of `fun` that don't have a default value"?

Reply:
This gets changed again in #131, so I don't think we need to worry about it here.